.gitignore

```diff
@@ -179,3 +179,5 @@ tramp
 # cask packages
 .cask/
 
+node_modules/
+TWLight/static/css/*-rtl.css
```

Comment:
we need the rtl css files in version control for production deployment; git ignoring node_modules makes sense though.

Reply:
Interesting, my concern is that if we change new-local.css and then forget to update the corresponding RTL css file then there could be drift between the two.

I kind of like the idea of having a source of truth, that is then "transformed" to generate the RTL css if that makes sense. 
I also assumed that since new-local-rtl.css is not being checked into git and is generated we would want to do the same for the rest of the -rtl.css files, but definitely correct me if I'm wrong. 

I also have little to no context around how deploys to prod go, so take my input with a grain of salt.